### PR TITLE
Add InputEvent documentation

### PR DESCRIPTION
**Pull request by alucas**

### CODE REVIEW

File: src/main/java/net/minecraftforge/client/event/InputEvent.java [hunk 1]
```diff
@@ -42,7 +42,7 @@ public MouseInputEvent(int button, int action, int mods)
         }
 
         /**
-         * The mouse button that was pressed or released
+         * The mouse button triggered this event.
```
Comment (Gamebuster19901) [inline src/main/java/net/minecraftforge/client/event/InputEvent.java]: This is missing the world 'that' between 'button' and 'triggered'

File: src/main/java/net/minecraftforge/client/event/InputEvent.java [hunk 2]
```diff
@@ -27,7 +27,7 @@
 public class InputEvent extends Event
 {
     /**
-     * This event fires when a mouse button is pressed.
+     * Event fired when a mouse input is detected.
```
Comment (Gamebuster19901) [inline src/main/java/net/minecraftforge/client/event/InputEvent.java]: I think we should stick with "This event fires when..."

File: src/main/java/net/minecraftforge/client/event/InputEvent.java [hunk 3]
```diff
@@ -79,7 +79,7 @@ public int getMods()
     }
 
     /**
-     * This event fires when keyboard input is detected.
+     * Event fired when a keyboard input is detected.
```
Comment (Gamebuster19901) [inline src/main/java/net/minecraftforge/client/event/InputEvent.java]: I think we should stick with "This event fires when..."

**Review by Gamebuster19901 — COMMENTED** [review]
There are some minor changes I'd make, but overall seems decent enough.

**Review by Gamebuster19901 — APPROVED** [review]

**Review by mezz — APPROVED** [review]

**Pull request merged**